api/utils/addr.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package webclient provides a client for the Teleport Proxy API endpoints.
+package utils
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+
+	"github.com/gravitational/trace"
+)
+
+// ParseAddr strings like "tcp://host:port/path" and returns "host:port".
+func ParseAddr(addr string) (string, error) {
```

Comment:
I think this function should be named more distinctively as there already is a `lib/utils.ParseAddr` which fits better. What about `ParseHostPort` or `ExtractHostPort` (in which case it could be sensible to omit the unix scheme entirely)?